src/amitaRJun20/SumOfAllNumbersInString.java

```diff
@@ -0,0 +1,30 @@
+/*Assignment 25 : 8th July 2020
+
+WAP to sum all the numbers in given string.
+Input: J7yu9y8h1h8j4b7j3jjb6
+Ouput: Sum of all numbers in String is: 54*/
+package amitaRJun20;
+
+import java.util.Arrays;
+
+public class SumOfAllNumbersInString {
+	
+	int sum = 0;
+	
+	//method to display sum of all numbers in input String
+	void displaySum(String input) {
+		for (int index = 0; index < input.length(); index++) {
+			if (Character.isDigit(input.charAt(index))) {
+				int number = Integer.parseInt(String.valueOf(input.charAt(index)));
```

Comment:
We can avoid creating number variable and directly add this value using the expression

Reply:
updated as per suggestion...Thank you for feedback..